configuration.go

```diff
@@ -270,6 +272,12 @@ func NewTraefikDefaultPointersConfiguration() *TraefikConfiguration {
 	defaultDocker.ExposedByDefault = true
 	defaultDocker.Endpoint = "unix:///var/run/docker.sock"
 
+	//default Swarm
+	var defaultSwarm provider.Swarm
+	defaultSwarm.Watch = true
+	defaultSwarm.ExposedByDefault = true
+	defaultSwarm.Endpoint = "unix:///var/run/docker.sock"
```

Comment:
@timchunght `/var/run/docker.sock` or `tcp://<ip>:2375` are use to access the docker remote api, when you set up the swarm mode you have 2 types of nodes, manages and works, so the works nodes don't accept remote API calls. This PR was a code that  traefik data get from services instead containers.(Sorry my bad English 😄 )